### ISSUE TITLE
When I using my data ,I got some error on it. Can you help me?

### ISSUE THREAD

**Issue by zoanana990** [armaanpriyadarshan/Training-a-Custom-TensorFlow-2.X-Object-Detector#17]
When I run example for pill data ,it can work successfully. But when I run for my own data , it give me some error.
![image](https://user-images.githubusercontent.com/63161051/103509497-3ec79a80-4e9e-11eb-901a-6eb0669494e3.png)
The only difference between data is the size ,but when I resize the data ,I still got this error .
Can you help me to fix it QAQ

**Comment by mahamatnoumai** [comment]
@zoanana990 can you try to use different batch size and see if your datasets have bigger size.

**Comment by zoanana990** [comment]
@mahamatnoumai Did you mean the file _pipeline.config_ line135 batch_size = 6 ? I had tried 1, 6, 10 . Unfortunately, it still not work. Still have same error QAQ
![image](https://user-images.githubusercontent.com/63161051/103520472-2d888900-4eb2-11eb-940c-113277cc03fc.png)

**Comment by mahamatnoumai** [comment]
@zoanana990 Yeah, if that doesn't solve ur issues.  could you please try to redo all the steps over again, and make sure all configuration setup were done correctly, then it may run smoothly. and if it doesn't work in your second trial, then share your config file as well, we could investigate, and may suggest some ideas.

**Comment by armaanpriyadarshan** [comment]
Yes, what @mahamatnoumai is correct. Did you try reconfiguring?
